src/configs/base.ts

```diff
@@ -208,6 +208,15 @@ export const config: Config = {
     },
   ],
   onSurface: [
+    {
+      name: 'background',
+      description: 'For use as a background color on components',
```

Comment:
```suggestion
      description: 'For use as a background color, in components on surface elements such as SearchField',
```

I'm going back on what I said after realizing we mention components elsewhere in the docs. I'd just be a little more specific and add that it's the background for components overlaid on surface colored elements. Sorry 😄